source/MaterialXRenderGlsl/GlslRenderer.cpp

```diff
@@ -348,11 +351,26 @@ void GlslRenderer::drawScreenSpaceQuad()
     glBindBuffer(GL_ARRAY_BUFFER, vbo);
     glBufferData(GL_ARRAY_BUFFER, sizeof(QUAD_VERTICES), QUAD_VERTICES, GL_STATIC_DRAW);
 
-    glEnableVertexAttribArray(0);
-    glVertexAttribPointer(0, 3, GL_FLOAT, GL_FALSE, 5 * sizeof(float), (void*) 0);
-
-    glEnableVertexAttribArray(1);
-    glVertexAttribPointer(1, 2, GL_FLOAT, GL_FALSE, 5 * sizeof(float), (void*) (3 * sizeof(float)));
+    for (auto input: _program->getAttributesList())
+    {
+        if (input.first.find(HW::IN_POSITION) != std::string::npos)
+        {
+            glEnableVertexAttribArray(input.second->location);
+            glVertexAttribPointer(input.second->location, 3, GL_FLOAT, GL_FALSE, stride * sizeof(float), (void*) 0);
+        }
+        
+        if (input.first.find(HW::IN_NORMAL) != std::string::npos)
```

Comment:
Do these constant normals have an effect on the rendered output?  Since you're omitting them in the Viewer implementation, perhaps we can remove them here as well.

Reply:
Removed for now.